task/02-numbers-tasks.js

```diff
@@ -221,7 +231,7 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    return !isNaN(parseFloat(value)) && isFinite(value) ? value : def;
```

Comment:
`isFinite` is not a safe method. Use `Number.isFinite` instead.

Reply:
You can use unary `+` to convert a string to number instead of `parseFloat(value)`.